common/lib/xmodule/xmodule/course_module.py

```diff
@@ -183,6 +183,15 @@ class CourseFields(object):
                  default=DEFAULT_START_DATE,
                  scope=Scope.settings)
     end = Date(help="Date that this class ends", scope=Scope.settings)
+    cosmetic_display_price = Integer(
+        display_name=_("Cosmetic Course Display Price"),
+        help=_(
+            "The cost displayed to students for enrolling in the course. If a paid course registration price is "
+            "set by platform staff in the database, that price will be displayed instead of this one."
```

Comment:
is "platform staff" a term we use anywhere else? 

Maybe we use like, "administrator" or something?